### ISSUE TITLE
perl warning about locales

### ISSUE THREAD

**Issue by sbambrough** [96boards/bugs#52]
linaro@linaro-alip:~$ sudo apt-get install dbus-x11
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following NEW packages will be installed:
  dbus-x11
0 upgraded, 1 newly installed, 0 to remove and 22 not upgraded.
Need to get 71.5 kB of archives.
After this operation, 156 kB of additional disk space will be used.
Get:1 http://http.debian.net/debian/ jessie/main dbus-x11 arm64 1.8.16-1 [71.5 kB]
Fetched 71.5 kB in 1s (53.4 kB/s)   
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = (unset),
	LC_ALL = (unset),
	LC_TIME = "en_CA.UTF-8",
	LC_MONETARY = "en_CA.UTF-8",
	LC_ADDRESS = "en_CA.UTF-8",
	LC_TELEPHONE = "en_CA.UTF-8",
	LC_NAME = "en_CA.UTF-8",
	LC_MEASUREMENT = "en_CA.UTF-8",
	LC_IDENTIFICATION = "en_CA.UTF-8",
	LC_NUMERIC = "en_CA.UTF-8",
	LC_PAPER = "en_CA.UTF-8",
	LANG = "C.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to a fallback locale ("C.UTF-8").
locale: Cannot set LC_ALL to default locale: No such file or directory
Selecting previously unselected package dbus-x11.
(Reading database ... 54229 files and directories currently installed.)
Preparing to unpack .../dbus-x11_1.8.16-1_arm64.deb ...
Unpacking dbus-x11 (1.8.16-1) ...
Setting up dbus-x11 (1.8.16-1) ...

Is there any way to get rid of the warning from perl regarding the locales?

**Comment by fboudra** [comment]
the warning is harmless. the locales aren't configured by default.

**Comment by suihkulokki** [comment]
Scott,

I don't think building all locales anyone in linaro might use makes sense. Either unset your locale before ssh, or can build your own locales on the board using:

dpkg-reconfigure locales

**Comment by sbambrough** [comment]
I agree.  However we should make this an FAQ item so we have a stock answer.

**Comment by sbambrough** [comment]
I needed to do the following to fix this issue:
sudo apt-get install locales
sudo dpkg-reconfigure locales
